### PR TITLE
dev-{haskell,java,lang}/: Fix toolong DESCRIPTION

### PR DESCRIPTION
**Pull request by Moiman**

### CODE REVIEW

**Comment by SoapGentoo** [comment]
@chewi and @trofi ACK?

File: dev-java/dynalang/dynalang-0.4-r1.ebuild [hunk 1]
```diff
@@ -10,7 +10,7 @@ inherit java-pkg-2 java-ant-2
 
 MY_P="${PN}-mop-${PV}"
 
-DESCRIPTION="Library for implementation of interoperable metaobject protocols for dynamic languages"
+DESCRIPTION="Implementation of interoperable metaobject protocols for dynamic languages"
```
Comment (chewi) [inline dev-java/dynalang/dynalang-0.4-r1.ebuild]: These descriptions aren't equivalent. I'm not sure what I'd put instead but I think "library" needs to be in there somewhere.

File: dev-java/sblim-cim-client/sblim-cim-client-2.2.1-r1.ebuild [hunk 2]
```diff
@@ -7,7 +7,7 @@ JAVA_PKG_IUSE="doc source test"
 
 inherit java-pkg-2 java-ant-2 eutils
 
-DESCRIPTION="A WBEM services client that includes an IETF RFC 2614 compliant SLP client for CIM service discovery"
+DESCRIPTION="A WBEM services client that includes a SLP client for CIM service discovery"
```
Comment (chewi) [inline dev-java/sblim-cim-client/sblim-cim-client-2.2.1-r1.ebuild]: …a**n** SLP…

File: dev-java/sblim-cim-client/sblim-cim-client-2.2.1.ebuild [hunk 3]
```diff
@@ -7,7 +7,7 @@ JAVA_PKG_IUSE="doc source test"
 
 inherit java-pkg-2 java-ant-2
 
-DESCRIPTION="A WBEM services client that includes an IETF RFC 2614 compliant SLP client for CIM service discovery"
+DESCRIPTION="A WBEM services client that includes a SLP client for CIM service discovery"
```
Comment (chewi) [inline dev-java/sblim-cim-client/sblim-cim-client-2.2.1.ebuild]: …an SLP…

**Comment by monsieurp** [comment]
OK @chewi's comments have been addressed. It LGTM too. This PR is ready to go.

File: dev-java/dynalang/dynalang-0.4-r1.ebuild [hunk 4]
```diff
@@ -10,7 +10,7 @@ inherit java-pkg-2 java-ant-2
 
 MY_P="${PN}-mop-${PV}"
 
-DESCRIPTION="Library for implementation of interoperable metaobject protocols for dynamic languages"
+DESCRIPTION="Libary of interoperable metaobject protocols for dynamic languages"
```
Comment (chewi) [inline dev-java/dynalang/dynalang-0.4-r1.ebuild]: Typo!

**Comment by monsieurp** [comment]
Sorry @chewi, I saw your comment after merging. Typo's been fixed in a separate commit though, no worries. :+1:

**Pull request closed without merging**